dataset/__main__.py

```diff
@@ -15,7 +15,16 @@
 
 import uvicorn
 
-from dataset.main import app  # noqa
+from dataset.config import get_settings
 
+settings = get_settings()
 if __name__ == '__main__':
-    uvicorn.run('run:app', host='127.0.0.1', port=5081, log_level='info', reload=True)
+    uvicorn.run(
+        'dataset.app:create_app',
+        factory=True,
+        host=settings.HOST,
+        port=settings.PORT,
+        workers=settings.WORKERS,
+        log_level=settings.LOG_LEVEL_DEFAULT,
+        reload=True,
```

Comment:
I would suggest to add `RELOAD` option into settings.
An example:  https://github.com/PilotDataPlatform/notification/blob/937b79f09d51939f2b304be5625fe49fc621105f/notification/config.py#L60

```suggestion
        reload=settings.RELOAD,
```

We shouldn't run app with hardcoded `reload=True` in production.